src/cpu/slime_check/scalar.zig

```diff
@@ -0,0 +1,103 @@
+const std = @import("std");

```

Comment:
this whole file can just be in `SearchBlock.zig`, it's not big enough or distinct enough to be its own file

Reply:
IMO it's beneficial to have distinct files for the SIMD and scalar implementations for organizational purposes